stable/mariadb/README.md

```diff
@@ -43,122 +43,93 @@ The command removes all the Kubernetes components associated with the chart and
+The following tables lists the configurable parameters of the MariaDB chart and their default values.
+
+|             Parameter                     |                     Description                     |                              Default                              |
+|-------------------------------------------|-----------------------------------------------------|-------------------------------------------------------------------|
+| `image.registry`                          | MariaDB image registry                              | `docker.io`                                                       |
+| `image.repository`                        | MariaDB Image name                                  | `bitnami/mariadb`                                                 |
+| `image.tag`                               | MariaDB Image tag                                   | `{VERSION}`                                                       |
+| `image.pullPolicy`                        | MariaDB image pull policy                           | `Always` if `imageTag` is `latest`, else `IfNotPresent`           |
+| `image.pullSecrets`                       | Specify image pull secrets                          | `nil` (does not add image pull secrets to deployed pods)          |
+| `service.type`                            | Kubernetes service type                             | `ClusterIP`                                                       |
+| `service.port`                            | MySQL service port                                  | `3306`                                                            |
+| `root.password`                           | Password for the `root` user                        | _random 10 character alphanumeric string_                         |
+| `db.user`                                 | Username of new user to create                      | `nil`                                                             |
+| `db.password`                             | Password for the new user                           | _random 10 character alphanumeric string if `db.user` is defined_ |
+| `db.name`                                 | Name for new database to create                     | `my_database`                                                     |
+| `replication.enabled`                     | MariaDB replication enabled                         | `true`                                                            |
+| `replication.user`                        | MariaDB replication user                            | `replicator`                                                      |
+| `replication.password`                    | MariaDB replication user password                   | _random 10 character alphanumeric string_                         |
+| `master.antiAffinity`                     | Master pod anti-affinity policy                     | `soft`                                                            |
```

Comment:
Please let the user set the entire affinity block (like the mongodb-replicaset Chart, but most other that allow affinity to be set as well) as it's more flexible.